web/html/src/manager/systems/ssm/ssm-subscribe-channels.tsx

```diff
@@ -406,7 +406,7 @@ class ChildChannelPage extends React.Component<ChildChannelProps, ChildChannelSt
   };
 
   render() {
-    const rows = this.props.childChannels.map((allowed) => {
+    const rows = this.props.childChannels.map((allowed: SsmAllowedChildChannelsDto) => {
```

Comment:
Here and below, this manual annotation is not really necessary since the type is already inferred from the prop type.